.cirrus.yml

```
@@ -227,6 +227,7 @@ codecov_task:
     pip install pytest-cov codecov
 
   check_requirements_script:
+    check-manifest
```

This should fix the failure, based on the above syntax:

```suggestion
  check_requirements_script: |
    check-manifest
```

---

thanks that fixed it